akka-typed/src/main/scala/akka/typed/internal/ActorCell.scala

```diff
@@ -326,6 +315,23 @@ private[typed] class ActorCell[T](
         else unsafe.getAndAddInt(this, statusOffset, -1)
       }
     }
+
+    try {
+      unscheduleReceiveTimeout()
+      if (!isTerminated(status))
+        process()
+      scheduleReceiveTimeout()
+      afterProcess()
+    } catch {
+      case NonFatal(ex) ⇒
+        fail(ex)
+        afterProcess()
```

Comment:
I don't want to all that stuff in the finally when `OutOfMemory` (or similar) is thrown